plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/index.tsx

```diff
@@ -29,6 +31,14 @@ if ( shouldRegisterBlock ) {
 	// Register the store
 	registerStore();
 
+	// loads the product types
+	dispatch( woocommerceTemplateStateStore ).setProductTypes(
+		getProductTypeOptions()
+	);
+
+	// Select Simple product type
+	dispatch( woocommerceTemplateStateStore ).switchProductType( 'simple' );
```

Comment:
Just thought about this, but given that the product type list [is filterable by extensions](https://github.com/woocommerce/woocommerce/blob/d05b77e66bd35231cd2c82fd87c96ce4f1dba970/plugins/woocommerce/includes/wc-product-functions.php#L640), there might be the chance that they removed `simple`. I wonder if instead of defaulting to `simple` we should default to the first product type in the list (in most stores it will still be simple, but if they removed it from the array, it will pick the next one). What do you think?

Reply:
what happens if the first one is not the Simple product? 😬 We can find the `simple` slug among all products. If it isn't found, then we can pass the first one.

Reply:
To me, using the first one by default would be the best approach. We will need to test with popular extensions to make sure they don't add their products types before `simple` unintentionally, but as long as that doesn't happen, it feels the most correct approach to me.

So if a store wants to have a different product type as the first one, I think we should honor that.